src/coreclr/tools/aot/ILCompiler.TypeSystem.Tests/TypeEquivalenceAssembly/TypeEquivalenceAssembly2.csproj

```diff
@@ -11,7 +11,6 @@
     <RuntimeMetadataVersion>v4.0.30319</RuntimeMetadataVersion>
     <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
     <RunAnalyzers>false</RunAnalyzers>
-    <DefineConstants>TYPEEQUIVALENCEASSEMBLY_2</DefineConstants>
```

Comment:
Unused

Reply:
the same one is in src/tests/baseservices/typeequivalence/istypeequivalent/typeequivalenttypes_2.csproj, we can remove it there too